### ISSUE TITLE
I can and want to help expand HotVoice with Korean language.

### ISSUE THREAD

**Issue by PurplePaperPlane** [evilC/HotVoice#6]
Same as the title. I'm a native Korean speaker and I voluntary want to help to expand HotVoice with it.

p.s. I'm not that good on codes or something but well, if I can help even with this condition, I would help!

**Comment by evilC** [comment]
Hi,
The email notification that I received has a Korean version of the `Simple Example` script in it, but I don't see that here?
Are there any changes needed to the library for it to work with Korean?
If not, I guess it would make sense to alter the main `Demo.ahk` file (The one that demonstrates multiple languages) to include Korean?
It would be pretty cool to have an example of non-english character sets in the demo to prove that it still works

**Comment by PurplePaperPlane** [comment]
Eh, actually it does not need any kind of patch or something, just to change the encoding form of the file to UTF 8 BOM.
and here is the Simple Example.ahk translated to Korean.

I don't know if that's possible to apply to `Demo.ahk`. Because if the script file is not encoded with UTF 8 BOM, Korean letters looks weird. I have attached an example ahk that shows korean letters error.

[Simple Example - ko.zip](https://github.com/evilC/HotVoice/files/9299931/Simple.Example.-.ko.zip
![a](https://user-images.githubusercontent.com/84488022/183900786-eef17ee5-bf4f-44c3-81be-3c5cd0a2f1dd.png)
)

**Comment by PurplePaperPlane** [comment]
and.... here goes the `Demo.ahk` with Korean option added. 
I have tested it, it works nicely. But sometimes it can not recognation 'percent's well. for an Example I said '볼륨 95 퍼센트'(Volume 95 percent) but it detects it like 91 sometimes.

and just like I said before, it needs to be encoded with UTF 8 BOM. The attached ahk example file below, is already encoded with it.

[Demo - ko.zip](https://github.com/evilC/HotVoice/files/9300117/Demo.-.ko.zip)
![b](https://user-images.githubusercontent.com/84488022/183905798-4280b2b9-8d33-4ca6-95f0-ddc0f6c11962.png)

**Comment by evilC** [comment]
OK, so opening this demo file, it seems like even plain old Notepad supports UTF-8 BOM
Could you maybe re-package it as `DemoInclude_ko.ahk` (As I have done with `DemoInclude_en.ahk`) and then submit a pull request?
If it's not recognizing the percents wholly reliably, I don't really consider that an issue - if it works at all, then the problem is most likely to do with Microsoft's recognizer, so not a lot I can do about that

**Comment by PurplePaperPlane** [comment]
I'm sorry that I can not understand how to use the pull request you told. I'm new on github and not got that much of info about this kind of works. I thought just you to add it to a next version of HotVoice's examples. :/

**Comment by evilC** [comment]
I am happy to do it myself if you want, but I would very much recommend you doing it yourself if you are interested in getting involved in open source projects.
The simplest way is to install GitHub Desktop, then on the main HotVoice page, click the green `Code` button and select `Open with Github Desktop`. This will result in you "cloning" a copy of the HotVoice source to your hard disk
Then, make the changes, and once you are done, commit and push them.
Once that is done, you use GitHub Desktop to create a "pull request", which I will receive, and once I approve it, it gets merged into the HotVoice repo and **you are credited as the author of that code**.

**Comment by PurplePaperPlane** [comment]
Thank you for the advice, but I do this purely to expand HotVoice not for my reputation. I do not need to be creditted. So please proceed without me.
and have a nice day. :D

also thanks for your great DLL!
